core/src/test/java/uk/ac/bbsrc/tgac/miso/core/data/workflow/impl/TestWorkflow.java

```diff
@@ -98,28 +123,32 @@ public WorkflowStepPrompt getPrompt() {
 
     @Override
     public String getLogMessage() {
-      return String.format("Processed Pool with id %d", progressStep.getInput().getId());
+      return String.format("Processed Pool with id %d", poolProgressStep.getInput().getId());
```

Comment:
Log messages should be more descriptive and specific to the workflow. e.g. "Selected Pool {alias} ({name})"

Prompt message should be similar. How and why is the user "inputting a pool?"